docs/tools/vega-wallet/desktop-app/index.md

```diff
@@ -3,6 +3,9 @@ title: Desktop Wallet
 hide_title: false
 ---
 
-The Vega Wallet is available to use as a desktop application. It allows you to use your existing Vega wallet, create new wallets, or restore wallets. The desktop app also lets you connect to Vega networks and start Vega dApps through the visual interface.
+The Vega Wallet is available to use as a desktop application. 
```

Comment:
suggestion (non-blocking): delete this line completely. rationale: Docs are not news/announcements. Particularly given the version selector.

Reply:
Interesting ... so the point of that is to help people differentiate between the desktop wallet and the CLI - describing what it's useful for. Do you think it does a bad job of that?

Reply:
I think the titles "CLI Wallet app" and "Desktop Wallet app" do a good enough job. But I don't think the line does any harm, just one that I think could be trimmed to cut down.